src/aioamazondevices/api.py

```
@@ -1181,51 +1211,46 @@ async def _send_message(
-        await self._session_request(
-            method=HTTPMethod.POST,
-            url=f"https://alexa.amazon.{self._domain}/api/behaviors/preview",
-            input_data=node_data,
-            json_data=True,
-        )
-
-        return
+        async with self._sequence_lock:
+            self._sequence_buffer.append(
+                {
+                    "@type": "com.amazon.alexa.behaviors.model.OpaquePayloadOperationNode",  # noqa: E501
```

Ruff (0.14.0)

Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)